src/main/java/br/com/caelum/carangobom/usuario/UsuarioController.java

```diff
@@ -31,4 +38,19 @@
             return ResponseEntity.badRequest().build();
         }
     }
+
+    @GetMapping
+    public ResponseEntity<List<UsuarioResponse>> getAll() {
+        return ResponseEntity.ok(usuarioService.findAll());
+    }
+
+    @DeleteMapping("/{id}")
+    public ResponseEntity<Void> delete(@Validated @PathVariable Long id) {
+        try {
+            usuarioService.delete(id);
+            return ResponseEntity.ok().build();
```

Comment:
Talvez seja legal para manter o padrão
```suggestion
            return ResponseEntity.noContent().build();
```

Reply:
Pois é, isso até ia conversar com vc msm, qual seria o melhor retorno nesse caso, porque no Marca usa o ok. Acho que o noContent é melhor ne?

Reply:
Se só está retornando o status, sem conteúdo, acho que faz mais sentido sim. (https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/DELETE)